internal/core/graphics/brush.rs

```diff
@@ -254,20 +260,173 @@ impl RadialGradientBrush {
 /// The ConicGradientBrush describes a way of filling a shape with a gradient
 /// that rotates around a center point
 #[derive(Clone, PartialEq, Debug)]
-#[repr(transparent)]
-pub struct ConicGradientBrush(SharedVector<GradientStop>);
+#[repr(C)]
+pub struct ConicGradientBrush {
+    /// The starting angle (rotation) of the gradient in normalized form (0.0 = 0°, 1.0 = 360°)
+    from_angle: f32,
+    /// The color stops of the gradient
+    stops: SharedVector<GradientStop>,
+}
 
 impl ConicGradientBrush {
-    /// Creates a new conic gradient with the provided color stops.
-    /// The stops should have angle positions in the range 0.0 to 1.0,
-    /// where 0.0 is 0 degrees (north) and 1.0 is 360 degrees.
-    pub fn new(stops: impl IntoIterator<Item = GradientStop>) -> Self {
-        Self(stops.into_iter().collect())
+    /// Creates a new conic gradient with the provided starting angle and color stops.
+    ///
+    /// The `from_angle` parameter is in normalized form (0.0 = 0°, 1.0 = 360°), corresponding
+    /// to CSS's `from <angle>` syntax. It rotates the entire gradient clockwise.
+    pub fn new(from_angle: f32, stops: impl IntoIterator<Item = GradientStop>) -> Self {
```

Comment:
For consistency with LinearGradientBrush::new, i think the angle should be specified in degrees. 
In Slint, angles are always degrees.